Makefile

```
@@ -47,3 +47,22 @@ seed:
 debug.smithd:
 	cargo build --release -p smith
 	sudo ln -sf $(CURDIR)/target/release/smithd /usr/bin/smithd
+
+DEVICES ?= $(shell docker ps --filter "name=smith-device" --format "{{.Names}}")
+
+watch.smithd:
+	cargo watch -s "make deploy.smithd" -w smithd -w models
+
+deploy.smithd:
```

checkmake (0.3.2)

[warning] Target body for "deploy.smithd" exceeds allowed length of 5 lines (12).

(maxbodylength)